test/integration/targets/vmware_inventory/runme.sh

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/env bash
+
+[[ -n "$DEBUG" || -n "$ANSIBLE_DEBUG" ]] && set -x
+
+set -euo pipefail
+
+export ANSIBLE_CONFIG=ansible.cfg
+export vcenter_host="${vcenter_host:-0.0.0.0}"
+export vcenter_user="${vcenter_user:-user}"
+export vcenter_pass="${vcenter_pass:-pass}"
+VMWARE_CONFIG=test-config.vmware.yaml
+
+cat > "$VMWARE_CONFIG" <<VMWARE_YAML
+plugin: vmware_inventory
+strict: False
+hostname: ${vcenter_host}
+username: ${vcenter_user}
+password: ${vcenter_pass}
+validate_certs: False
+with_tags: False
+VMWARE_YAML
+
+echo "Installing Pyvmomi"
+${ANSIBLE_TEST_PYTHON_INTERPRETER} -m pip install pyvmomi
```

Comment:
This is unnecessary since `pyvmomi` is automatically installed for all `cloud/vcenter` tests.